app/views/users/show.html.erb

```diff
@@ -1,38 +1,40 @@
 <section class="dashboard-main">
   <h1> <%= current_user.first_name %>'s Dashboard </h1>
-
   <%= button_to 'Log Out', logout_path, method: 'delete', class: "btn btn-primary mb1 bg-teal" %>
   <h3>Account Details</h3>
   <ul>
     <li> <%= current_user.first_name  %> <%= current_user.last_name %> </li>
     <li> <%= current_user.email%> </li>
   </ul>
-
-  <section>
-    <h1>Bookmarked Segments</h1>
-  </section>
 </section>
 
+<%= button_to "Connect to Github", "/auth/github" %>
+
 <% if current_user.token != nil %>
   <h2> Github </h2>
     <section id='repos'>
       <h3> Repositories </h3>
       <% current_user.gh_repos.each do |repo| %>
-        <%= link_to repo.name, repo.html_url %>
+        <%= link_to repo.name, repo.html_url %> <br>
```

Comment:
We can look at just making these their own css classes/tags later on. This will eliminate the need for the `<br>` tag at the end there.